config/routes.rb

```diff
@@ -1,7 +1,7 @@
 Rails.application.routes.draw do
-  get '/restaurant', controller: :restaurant, action: :index
+  get '/restaurant', controller: :restaurants, action: :index
```

Comment:
Okay, so I would like you to discuss this route with @JesperGreen I need you guys to have this as `root_path` Can you make sure that in between the two of you who will include that in a PR.